packages/bpk-component-modal/src/BpkModalInner.tsx

```diff
@@ -108,71 +121,33 @@ const BpkModalInner = (props: Props) => {
                   id={headingId}
                   className={getClassName('bpk-modal__heading')}
                 >
-                  {props.title}
+                  {title}
                 </h2>
               }
               leadingButton={accessoryViewFinal}
               trailingButton={
-                props.closeText ? (
+                closeText ? (
                   <BpkButtonLink
                     className={getClassName('bpk-modal__close-button')}
-                    onClick={props.onClose}
+                    onClick={onClose}
                   >
-                    {/* $FlowIgnore[incompatible-type] this is perfectly good because we're already doing a null check above. THANKS FLOW */}
-                    {props.closeText}
+                    {closeText}
                   </BpkButtonLink>
                 ) : (
                   <BpkCloseButton
                     className={getClassName('bpk-modal__close-button')}
-                    label={props.closeLabel}
-                    onClick={props.onClose}
+                    label={closeLabel}
+                    onClick={onClose}
                   />
                 )
               }
             />
           </header>
         )}
-        <div className={contentClassNames.join(' ')}>{props.children}</div>
+        <div className={contentClassNames.join(' ')}>{rest.children}</div>
```

Comment:
Do we not need to define this above instead of pulling it out of `rest`? Or is this how required props are supposed to work?